Makefile

```diff
@@ -2,16 +2,15 @@
 #
 # SPDX-License-Identifier: Apache-2.0
 
-EXTENSION_PREFIX            := gardener-extension
-NAME_ADMISSION              := shoot-rsyslog-relp-admission
 NAME                        := shoot-rsyslog-relp
-REGISTRY                    := eu.gcr.io/gardener-project/gardener
-IMAGE_PREFIX                := $(REGISTRY)/extensions
+NAME_ADMISSION              := $(NAME)-admission
+IMAGE                       := eu.gcr.io/gardener-project/gardener/extensions/shoot-rsyslog-relp
 REPO_ROOT                   := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 HACK_DIR                    := $(REPO_ROOT)/hack
 VERSION                     := $(shell cat "$(REPO_ROOT)/VERSION")
 EFFECTIVE_VERSION           := $(VERSION)-$(shell git rev-parse HEAD)
-GOARCH                      ?= $(shell go env GOARCH)
+IMAGE_TAG                   := $(EFFECTIVE_VERSION)
+LD_FLAGS                    := "-w $(shell EFFECTIVE_VERSION=$(EFFECTIVE_VERSION) $(REPO_ROOT)/vendor/github.com/gardener/gardener/hack/get-build-ld-flags.sh k8s.io/component-base $(REPO_ROOT)/VERSION $(NAME))"
```

Comment:
Instead of NAME, Can we not use `$EXTENSION_PREFIX-$NAME`? I see that in other extensions we only use prefix, but name also could be added.

Reply:
I copied this also from the `registry-cache` extension: [ref](https://github.com/gardener/gardener-extension-registry-cache/blob/a78ccd730f9dea3b43c32bfa6f21d231bfcc4f8a/Makefile#L24)
TBH, I'm not 100% sure what should be the correct program name here, just using the extension prefix seems weird to me.

Reply:
I would prefer `gardener-extension-shoot-rsyslog-relp` to be printed when we call the `verflag.PrintAndExitIfRequested()` function rather than `shoot-rsyslog-relp`. But I leave the choice to you.

Reply:
Yeah, the main problem is that it's the same for the admission, but we have this "issue" everywhere. Anyway, I will use your suggestion and set it to `gardener-extension-shoot-rsyslog-relp`

Reply:
> it's the same for the admission, but we have this "issue" everywhere

True.